blueprint/src/chapters/chapter1/sheet1.tex

```diff
@@ -0,0 +1,23 @@
+\section{Defining the Group}
+
+\begin{definition}[Group]
```

Comment:
I'm not sure I'm a fan of the result generated by adding the optional argument (what's in the square brackets).